<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>195</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>195</width>
    <height>100</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Error</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error: Input Not Correct</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>50</y>
      <width>205</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
